app/index.html

```diff
@@ -3,9 +3,14 @@
     <head>
         <meta charset="UTF-8" />
         <meta http-equiv="Content-Security-Policy" content="script-src 'self'"/>
-        <link rel="stylesheet" href="../node_modules/highlight.js/styles/vs.css" />
-        <link rel="stylesheet" href="../node_modules/github-markdown-css/github-markdown.css" />
         <link rel="stylesheet" href="index.css" />
+        <style>
+            @import "../node_modules/github-markdown-dracula-css/github-markdown-dracula.css" screen and (prefers-color-scheme: dark);
+            @import "../node_modules/highlight.js/styles/dracula.css" screen and (prefers-color-scheme: dark);
+
+            @import "../node_modules/github-markdown-css/github-markdown.css" screen and (prefers-color-scheme: light);
+            @import "../node_modules/highlight.js/styles/vs.css" screen and (prefers-color-scheme: light);
```

Comment:
Did you consider the print feature? At first glance it does not look like that but I didn't try this yet.

Reply:
Yeah it doesn't work :/

Reply:
![image](https://user-images.githubusercontent.com/20479804/109031315-84455e80-76bc-11eb-938d-789f7f9aa593.png)

Is this okay?

Reply:
I'm assuming it should only print out in light theme to not waste ink, unless it's to be printed into a pdf, I will have to look into another method

Reply:
This has made some theming errors, need to fix
![image](https://user-images.githubusercontent.com/20479804/109035051-29ae0180-76c0-11eb-889f-2436ca6e7849.png)

Reply:
The PDF looks good in the screenshot.

> I'm assuming it should only print out in light theme to not waste ink, unless it's to be printed into a pdf, if so I will have to look into another method

I don't think that a dark theme makes any sense on paper. The tool cannot and probably should not distinguish between printing on PDF and actual printing. I think "dark PDFs" are quite uncommon.

Reply:
Okay cool! I'm going to have to add some js to index.html as I will be unable to load in the dracula highlight.js theme properly while importing the vs highlight.js theme as a default